testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -529,7 +517,6 @@
 end
 
 Given(/^I am on the patches page$/) do
-  step %(I am authorized)
```

Comment:
Does this work? Because visiting the direct link opens a new session that redirects you to the login page right?

Reply:
Yup, it works thanks to cookies